tests/e2e/test_api_xai_sanity.py

```diff
@@ -156,32 +162,53 @@ def test_inference_xai(self):
             train_task.train(dataset, trained_model, TrainParameters())
             save_model_data(trained_model, temp_dir)
 
-            # Infer torch model
-            detection_environment.model = trained_model
-            inference_task = DetectionInferenceTask(task_environment=detection_environment)
-            predicted_dataset = inference_task.infer(dataset.with_empty_annotations())
-
-            # Check saliency maps torch task
-            task_labels = trained_model.configuration.get_label_schema().get_labels(include_empty=False)
-            saliency_maps_check(predicted_dataset, task_labels, self.ref_raw_saliency_shapes[model_template.name])
-
-            # Save OV IR model
-            inference_task._model_ckpt = osp.join(temp_dir, "weights.pth")
-            exported_model = ModelEntity(None, detection_environment.get_model_configuration())
-            inference_task.export(ExportType.OPENVINO, exported_model)
-            os.makedirs(temp_dir, exist_ok=True)
-            save_model_data(exported_model, temp_dir)
-
-            # Infer OV IR model
-            load_weights_ov = osp.join(temp_dir, "openvino.xml")
-            detection_environment.model = read_model(
-                detection_environment.get_model_configuration(), load_weights_ov, None
-            )
-            task = OpenVINODetectionTask(task_environment=detection_environment)
-            predicted_dataset_ov = task.infer(
-                dataset.with_empty_annotations(),
-                InferenceParameters(is_evaluation=False),
-            )
+            from otx.api.entities.subset import Subset
+
+            for processed_saliency_maps, only_predicted in [[True, False], [False, True]]:
+                detection_environment, dataset = self.init_environment(hyper_parameters, model_template, 10)
+                inference_parameters = InferenceParameters(
```

Comment:
Maybe we can use `ExlpainParameters` instead of `InferenceParameters`

Reply:
This is a test for task.infer, so we use `InferenceParameters`.